packages/@aws-cdk/aws-neptune/README.md

```diff
@@ -42,6 +42,23 @@ const cluster = new neptune.DatabaseCluster(this, 'Database', {
 
 By default only writer instance is provisioned with this construct.
 
+## Create instance type from non built-in instance type or CloudFormation parameter
+
+```ts
+const instanceType = new cdk.CfnParameter(stack, 'NeptuneInstaneType', {
+  type: 'String',
+  allowedValues: [
+    'db.r5.xlarge',
+    'db.r5.2xlarge',
+    'db.r5.4xlarge',
+    'db.r5.8xlarge',
+    'db.r5.12xlarge',
+  ],
+  default: 'db.r5.8xlarge',
+});
+const instanceType = neptune.InstanceType.of(instanceType.valueAsString);
+```
+
```

Comment:
I think this is one of those cases where this is niche enough that we don't need to update the README. Let's remove this; I'll tag the PR as being ok without README updates.

```suggestion
```